### PR TITLE
Refactor bridge.py

### PR DESCRIPTION
**Pull request by ebadi**
**Description** 
Hard-coded values and adjustments are scattered in the bridge.py which makes understanding the logic very difficult.
Here are some examples:

- `throttle_manual_multiplier = 0.7 `
- `brake_manual_multiplier = 0.7`
- `steer_manual_multiplier = 45 * STEER_RATIO`
- `throttle_op = clip(sm['carControl'].actuators.accel / 1.6, 0.0, 1.0)`
- `brake_op = clip(-sm['carControl'].actuators.accel / 4.0, 0.0, 1.0)`
- `vc.throttle = throttle_out / 0.6`

I merged parallel variables `throttle_X`,  `brake_X ` and `steer_X` into a `TrottleBrakeSteer` class, and these variables are refactored:
  
`throttle_manual, brake_manual, steer_manual` :arrow_right:  `manual` object
`throttle_out,  brake_out, steer_out` :arrow_right: `out` object
`old_throttle, old_brake, old_steer` :arrow_right: `old` object

I moved all CARLA simulator  and manual driving adjustments to two separate functions `TBS_rate_limit` and `TBS_scale_clamp`.
(TBS~TrottleBrakeSteer) and as the result, I got rid of these variables:
- `REPEAT_COUNTER`
- `STEER_RATIO`
- `max_steer_angle`
- `throttle_ease_out_counter`
- `steer_manual_multiplier`
- `throttle_manual_multiplier`
- `brake_manual_multiplier`
- `steer_carla`

The goal is to make integration of other simulators and environments easier in the future by isolating CARLA specific calibrations.

**Verification**

The autopilot and manual driving is tested.

### CODE REVIEW

File: tools/sim/README.md [hunk 1]
```diff
@@ -58,3 +58,43 @@ All inputs:
...
+brake (float)
+A scalar value to control the vehicle brake [0.0, 1.0]. Default is 0.0. 
+steer (float)
+A scalar value to control the vehicle steering [-1.0, 1.0]. Default is 0.0.
+```
+* [car.capnp](https://github.com/commaai/cereal/blob/master/car.capnp#L334)
+
+```
+                  ┌──────────────────┐
+                  │ OpenPilot/Manual │
```
Comment (incognitojam) [inline tools/sim/README.md:77]:
```suggestion
                  │ openpilot/Manual │
```

**Review by ebadi — COMMENTED** [review]
Changes seem to be fine

**Comment by adeebshihadeh** [comment]
Thanks for the PR, but this work is also being done in #27373

**Comment by ebadi** [comment]
> Thanks for the PR, but this work is also being done in #27373

Sure, with #27373 this PR (which was opened 3 months ago) becomes obsolete. 

p.s. I took a look at the new PR and they did a good job of refactoring the code but it seems they are not addressing the issues above. Anyways maintaining and adjusting this PR on top of their PR is very difficult considering that #27373 is a very **large** pull request.

**Pull request closed without merging**